content/blog/posts/student-story-whitehet.rst

```diff
@@ -0,0 +1,41 @@
+Reflections on My Time at the Open Source Lab, Travis Whitehead
+---------------------------------------------------------------
+:author: Travis Whitehead
+:date: 2019-04-24
+:slug: student-story-whitehet
+:tag: student-stories
```

Comment:
Please change this to the following:
``` rst
:tag: featured-stories
:tags: student-stories
```

Can you also add an ``:img:`` here since this will be featured on the front page? It'll need to be formatted to fit our aspect ratio.

Reply:
I don't think I have any photos of just me that fit the aspect ratio (the headshot I got from Gale is more of a vertical photo). Could I use that group/team photo we took at Barcamp?

Reply:
I took some photos with Pam.

Reply:
Please make sure you optimize them for the web so the file size is small.

Reply:
This needs to be ``:tags: student-stories``